recipes/nng/all/conanfile.py

```diff
@@ -20,13 +20,15 @@ class NngConan(ConanFile):
         "nngcat": [True, False],
         "http": [True, False],
         "tls": [True, False],
+        "maxTaskqThreads": "ANY"
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "nngcat": False,
         "http": True,
         "tls": False,
+        "maxTaskqThreads": None
```

Comment:
Idea: if this option is not provided, the default value from the cmake file shall be taken.